src/desktop/OpenRAP/sdks/UserSDK.ts

```diff
@@ -148,7 +148,11 @@ export class UserSDK {
   }
 
   public async getUserSession() {
```

Comment:
No need to handle like this return the promise and handle at other ends with catch